### PR TITLE
feat: Add shell completions for Nushell and Elvish

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
# Description

- Nushell completions.
  _Uses the [clap_complete_nushell] crate, as linked by the already used [clap_complete] crate._
- Elvish completions.
  _Already included in [clap_complete], so why not?_ 
- Update xtasks to generate the new shell completions.
- New xtask command to run all xtasks (useful for local development).
- Disable shellcheck on generated completions.

[clap_complete_nushell]: https://crates.io/crates/clap_complete_nushell
[clap_complete]: https://crates.io/crates/clap_complete

### CODE REVIEW

File: rust/stackablectl/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Shell completions for Nushell and Elvish ([#337]).
```
Comment (Techassi) [inline rust/stackablectl/CHANGELOG.md]: suggestion:

```suggestion
- Add shell completions for Nushell and Elvish ([#337]).
```

File: extra/completions/stackablectl.elv
Comment (Techassi) [inline extra/completions/stackablectl.elv:1]: note: Just making sure we use the correct (by convention) file extension for this completion file.

File: extra/completions/stackablectl.nu
Comment (Techassi) [inline extra/completions/stackablectl.nu:1]: note: Just making sure we use the correct (by convention) file extension for this completion file.

File: rust/stackablectl/CHANGELOG.md (hunk 1 above)
Comment (NickLarsenNZ) [reply rust/stackablectl/CHANGELOG.md]: I felt it was redundant due to the `Added` heading. Will await your response before dong anything.

File: extra/completions/stackablectl.elv
Comment (NickLarsenNZ) [reply extra/completions/stackablectl.elv:1]: I did searches at the time and they appeared to be correct.

File: extra/completions/stackablectl.nu
Comment (NickLarsenNZ) [reply extra/completions/stackablectl.nu:1]: I did searches at the time and they appeared to be correct.

File: rust/stackablectl/CHANGELOG.md (hunk 1 above)
Comment (Techassi) [reply rust/stackablectl/CHANGELOG.md]: I usually still use a verb at the start of the entry. This is how most entries are done in the rest of the changelog.

**Review by Techassi — APPROVED** [review]

**Pull request merged**